Library/Formula/pmd.rb

```diff
@@ -1,8 +1,20 @@
 class Pmd < Formula
   desc "Source code analyzer for Java, JavaScript, and more"
   homepage "http://pmd.sourceforge.net/"
-  url "https://github.com/pmd/pmd/releases/download/pmd_releases/5.3.3/pmd-src-5.3.3.zip"
-  sha256 "38004ea3274d2f71701f438606a4c4095b7a86110e8d7f2a8940170c9bd5ddbb"
+  head "https://github.com/pmd/pmd.git"
+
+  stable do
+    url "https://github.com/pmd/pmd/releases/download/pmd_releases/5.4.1/pmd-src-5.4.1.zip"
+    sha256 "1dcef1a9f6e7681e474faf2d37ed82b408edf880b3707f17f42b0046cf3130a1"
+
+    # Some versions of PMD have incorrect versions specified for parent POM's.
```

Comment:
I'd probably prefer to just not update for now if it requires this and the weird XML parsing.